main.py

```diff
@@ -148,9 +148,11 @@ def upload(self, screenshot, name):
 						pass
 			source = tmpFilename
 			destination = self.folder + "/" + ScreenCloud.formatFilename(filename)
+			png_build = bytes()
```

Comment:
```suggestion
			buffer = bytes()
```

Reply:
Hi Olav! yep, it's much better now, thanks